riskified-sdk/src/main/java/com/riskified/RiskifiedClient.java

```diff
@@ -891,10 +891,15 @@ private HttpClient constructHttpClient() {
 		HttpClientBuilder builder = HttpClientBuilder.create();
 		builder.setDefaultRequestConfig(requestBuilder.build());
 
-		if (this.proxyUrl != null) {
+		if (this.proxyUrl != null && this.proxyUsername != null) {
 			setProxyWithAuth(builder);
 		}
 
+        if (this.proxyUrl != null && this.proxyUsername == null){
```

Comment:
Why are we checking for proxyUsername here? is this part of the old code?

Reply:
This is to check if the proxy url and password is passed in, the other is to check when no username and password is passed in